### PR TITLE
[Improve] Improve streampark-console core package module service remove method name base on [3.1 Naming Style]

### PR DESCRIPTION
**Pull request by VampireAchao**
## What changes were proposed in this pull request

Issue Number: see #3064  

## Brief change log

Improve streampark-console core package module service name base on [3.1 Naming Style]

1. Just remove method
2. Just in service

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

Yes *(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

No *(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): No

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/VariableService.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public interface VariableService extends IService<Variable> {
    *
    * @param variable the Variable object to be deleted
    */
-  void deleteVariable(Variable variable);
+  void removeVariable(Variable variable);
```
Comment (caicancai) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/VariableService.java]: removeByVariable or remove?

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/ApplicationLogService.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public interface ApplicationLogService extends IService<ApplicationLog> {
 
   IPage<ApplicationLog> getPage(ApplicationLog applicationLog, RestRequest request);
 
-  void removeApp(Long appId);
+  void removeByAppId(Long appId);
 
-  Boolean delete(ApplicationLog applicationLog);
+  boolean removeById(ApplicationLog applicationLog);
```
Comment (caicancai) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/ApplicationLogService.java]: removeById？There is no id in the parameters

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/FlinkEnvController.java [hunk 3]
```diff
@@ -96,7 +96,7 @@ public RestResponse update(FlinkEnv version) throws Exception {
   @Operation(summary = "Delete flink environment")
   @PostMapping("delete")
   public RestResponse delete(Long id) {
-    flinkEnvService.delete(id);
+    flinkEnvService.remove(id);
```
Comment (caicancai) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/FlinkEnvController.java]: removeById

**Comment by VampireAchao** [comment]
Done.

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/ApplicationBackUpService.java [hunk 4]
```diff
@@ -74,7 +74,7 @@ public interface ApplicationBackUpService extends IService<ApplicationBackUp> {
    *
    * @param appParam the application to be removed
    */
-  void removeApp(Application appParam);
+  void removeByApp(Application appParam);
```
Comment (caicancai) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/ApplicationBackUpService.java]: Some places are Boolean remove (Application appParam); Could you please check it out?

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/ApplicationManageService.java [hunk 5]
```diff
@@ -105,7 +105,7 @@ public interface ApplicationManageService extends IService<Application> {
    * @param appParam The Application to be deleted.
    * @return True if the deletion was successful, false otherwise.
    */
-  Boolean delete(Application appParam);
+  Boolean remove(Application appParam);
```
Comment (caicancai) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/ApplicationManageService.java:108]: this

**Comment by caicancai** [comment]
cc @RocMarshal

**Review by caicancai — APPROVED** [review]

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/ApplicationController.java [hunk 6]
```diff
@@ -344,15 +344,15 @@ public RestResponse deleteOperationLog(ApplicationLog applicationLog) {
   @PostMapping("delete")
   @RequiresPermissions("app:delete")
   public RestResponse delete(Application app) throws InternalException {
-    Boolean deleted = applicationManageService.delete(app);
+    Boolean deleted = applicationManageService.removeByApp(app);
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/ApplicationController.java]: remove ...

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/ApplicationController.java [hunk 7]
```diff
@@ -335,7 +335,7 @@ public RestResponse optionlog(ApplicationLog applicationLog, RestRequest request
   @PostMapping("deleteOperationLog")
   @RequiresPermissions("app:delete")
   public RestResponse deleteOperationLog(ApplicationLog applicationLog) {
-    Boolean deleted = applicationLogService.delete(applicationLog);
+    Boolean deleted = applicationLogService.removeById(applicationLog);
     return RestResponse.success(deleted);
   }
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/ApplicationController.java:340]: We can improve this method :
```
public RestResponse deleteOperationLog(Long id) {
    Boolean deleted = applicationLogService.removeById(id);
    return RestResponse.success(deleted);
  }
```

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/ApplicationLogService.java [hunk 8]
```diff
@@ -27,7 +27,7 @@ public interface ApplicationLogService extends IService<ApplicationLog> {
 
   IPage<ApplicationLog> getPage(ApplicationLog applicationLog, RestRequest request);
 
-  void removeApp(Long appId);
+  void removeByAppId(Long appId);
 
-  Boolean delete(ApplicationLog applicationLog);
+  boolean remove(ApplicationLog applicationLog);
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/ApplicationLogService.java]: We don't need this method.

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationLogServiceImpl.java [hunk 9]
```diff
@@ -50,14 +50,13 @@ public IPage<ApplicationLog> getPage(ApplicationLog applicationLog, RestRequest
...
-  public void removeApp(Long appId) {
+  public void removeByAppId(Long appId) {
     LambdaQueryWrapper<ApplicationLog> queryWrapper =
         new LambdaQueryWrapper<ApplicationLog>().eq(ApplicationLog::getAppId, appId);
     this.remove(queryWrapper);
   }
 
-  @Override
-  public Boolean delete(ApplicationLog applicationLog) {
+  public boolean remove(ApplicationLog applicationLog) {
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationLogServiceImpl.java]: We don't need this method.

**Comment by caicancai** [comment]
unreported exception InternalException; must be caught or declared to be thrown

**Comment by VampireAchao** [comment]
Resolved~

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**